saleor/payment/migrations/0048_populate_transaction_item_token.py

```diff
@@ -0,0 +1,27 @@
+from django.contrib.postgres.functions import RandomUUID
+from django.db import migrations
+
+
+# This data migration is handled in the 0043 with celery tasks.
+# 0043 is due to zero-downtime policy, this migration is to ensure
+# that we can change token to non-nullable.
+def populate_transaction_item_token(apps, schema_editor):
+    TransactionItem = apps.get_model("payment", "TransactionItem")
+
+    transactions = TransactionItem.objects.filter(token__isnull=True)
+    for transaction in transactions:
+        transaction.token = RandomUUID()
+
+    transactions.bulk_update(transactions, ["token"])
```

Comment:
Could be run in one update query
https://github.com/saleor/saleor/blob/main/saleor/checkout/migrations/0044_fulfill_checkout_line_new_fields.py#L15